### PR TITLE
Added question 7

### PR DESCRIPTION
**Pull request by alvinbengeorge**
check and merge

### CODE REVIEW

**Comment by sidhant-sriv** [comment]
Whoops, did the same question, I'll remove my question 7

**Comment by alvinbengeorge** [comment]
no its fine, yours is shorter lol

File: question 7.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+f=open('poem.txt','r').read()
```
Comment (sidhant-sriv) [inline question 7.py:1]: Upload poem.txt too, put the question in its folder with the text files

Comment (alvinbengeorge) [reply question 7.py:1]: oh cool

Comment (alvinbengeorge) [reply question 7.py:1]: I need to type it lmao, I used another sentence to check

Comment (sidhant-sriv) [reply question 7.py:1]: Copy it from the docx

Comment (alvinbengeorge) [reply question 7.py:1]: its a picture
right?

Comment (alvinbengeorge) [reply question 7.py:1]: https://products.groupdocs.app/viewer/view?file=881b5041-23db-480a-9dc3-c2114891b136/Grade%20XII%20Practical%20list%20-term%201.docx

Comment (sidhant-sriv) [reply question 7.py:1]: Check my commit, I copied it from the docx

Comment (alvinbengeorge) [reply question 7.py:1]: oh cool

**Comment by alvinbengeorge** [comment]
How do you edit a PR?
Biggie noob lol

**Comment by alvinbengeorge** [comment]
it does it on its own?

File: question 7.py (hunk 1 above)
Comment (alvinbengeorge) [reply question 7.py:1]: cool done

**Comment by sidhant-sriv** [comment]
> it does it on its own?

There is an edit option next to the name

**Comment by alvinbengeorge** [comment]
yes, I clicked that, but its only for editing title right?

**Pull request merged**